app/views/shelters/index.html.erb

```diff
@@ -2,5 +2,6 @@
 
 <% @shelters.each do |shelter| %>
   <h2><%= link_to shelter.name, "/shelters/#{shelter.id}" %></h2>
+  <%= link_to "Edit #{shelter.name}", "/shelters/#{shelter.id}/edit", method: "get"%>
```

Comment:
Was initially just an "Edit Shelter" name but needed to be specific to each shelter in order to test.